packages/botonic-core/src/utils.ts

```diff
@@ -36,26 +42,26 @@ export const isMobile = (mobileBreakpoint = 460) => {
   }
   return false
 }
-export function isFunction(o) {
+export function isFunction(o: any): boolean {
```

Comment:
Is this `any` or `string`? If you are restricting it to `string`, the compiler will force you to invoke it only when using `typeof`, but maybe it's not what you want here 🙏

Reply:
it should be any, but now that I have reviewed it, this function is very generic for what it's doing (write it down for next iterations). As here we are only checking if the `routes` defined by the developer are written as an array of routes or as a function returning this routes, maybe we could call it `isFunctionalRouter` or similar and passing the entire `routes` as argument.